internal/pkg/xds/adss/grpc_server.go

```diff
@@ -57,29 +49,28 @@ func NewServer(opts *ServerOpts, pushRequests <-chan xds.PushRequest, onNewSubsc
 		grpc:         grpcServer,
 		ads:          ads,
 		pushRequests: pushRequests,
-		opts:         opts,
 	}
 }
 
 // Run starts the gRPC server and the controllers.
 func (s *Server) Run(ctx context.Context) error {
 	var routinesGroup errgroup.Group
 
-	listener, err := net.Listen("tcp", fmt.Sprintf(":%d", s.opts.Port))
+	listener, err := net.Listen("tcp", ":15080")
```

Comment:
Probably not in the scope of the PR, but IIUC we are making it configurable through

https://github.com/openshift-service-mesh/federation/blob/62123baa1e0f4e67f77803aa244f4a7e717e5e6a/internal/pkg/config/federation_config.go#L19

https://github.com/openshift-service-mesh/federation/blob/62123baa1e0f4e67f77803aa244f4a7e717e5e6a/internal/pkg/config/federation_config.go#L71-L76

Should we capture it as an issue to ensure it's propagated?

Reply:
Please look at this comment: https://github.com/openshift-service-mesh/federation/pull/75#discussion_r1848126094.

Reply:
The reason why I brought it up is b/c I see this port showing up around [ZTunnel](https://github.com/search?q=repo%3Aistio%2Fistio+15080&type=issues), though I can't find anything significant in the docs.

Reply:
I was [prototyping](https://github.com/openshift-service-mesh/federation/compare/master...jewertow:federation:examples-ambient) initial ambient support and this port did not cause any issue.